rosie/build.gradle

```diff
@@ -18,26 +18,27 @@ apply plugin: 'com.android.library'
 apply from: 'https://raw.github.com/chrisbanes/gradle-mvn-push/master/gradle-mvn-push.gradle'
 
 android {
-  compileSdkVersion 23
-  buildToolsVersion "23.0.3"
+  compileSdkVersion projectCompileSdkVersion
+  buildToolsVersion projectBuildTools
 
   defaultConfig {
-    minSdkVersion 14
-    targetSdkVersion 23
+    minSdkVersion projectMinSdkVersion
+    targetSdkVersion projectTargetSdkVersion
   }
 }
 
 dependencies {
-  compile 'com.android.support:appcompat-v7:23.4.0'
-  compile 'com.android.support:recyclerview-v7:23.4.0'
+  implementation 'com.android.support:appcompat-v7:23.4.0'
+  implementation 'com.android.support:recyclerview-v7:23.4.0'
 
-  provided 'com.squareup.dagger:dagger-compiler:1.2.2'
+  annotationProcessor 'com.squareup.dagger:dagger-compiler:1.2.2'
+  compileOnly 'com.squareup.dagger:dagger-compiler:1.2.2'
```

Comment:
I don't know why the dagger dependency is added twice but I maintain them.

Reply:
I don't remember why xD